network-api/networkapi/mozfest/templates/mozfest/mozfest-base.html

```diff
@@ -19,6 +19,13 @@
     {% blocktrans %}Mozilla Festival{% endblocktrans %}{% if page.specifics.seo_title %} - {{ page.specifics.seo_title }}{% else %} - {{ page.title }}{% endif %}
 {% endblock %}
 
+{% block org_schema %}
+  {% if page.structured_data %}
+    <script type="application/ld+json">
```

Comment:
Is this actual json-ld, or is it just "json, who knows what kind"? Because if the latter, let's make this `type="application/json"` instead.